csvqb/csvqb/models/cube/catalog.py

```diff
@@ -2,23 +2,22 @@
 Catalog Metadata (base)
 -----------------------
 """
+from dataclasses import dataclass
 from datetime import datetime
-from typing import Optional, List
-from abc import ABC
+from typing import Optional
+from abc import ABC, abstractmethod
 
-from csvqb.models.validationerror import ValidationError
+from csvqb.models.pydanticmodel import PydanticModel
 
 
-class CatalogMetadataBase(ABC):
-    def __init__(
-        self,
-        title: str,
-        description: Optional[str] = None,
-        issued: Optional[datetime] = None,
-    ):
-        self.title: str = title
-        self.description: Optional[str] = description
-        self.issued: Optional[datetime] = issued
+@dataclass
+class CatalogMetadataBase(PydanticModel, ABC):
+    title: str
```

Comment:
Why are we not having the description or issued date in the base anymore?

Reply:
Because I ended up bashing my head against the desk.

Reply:
Too much pain. I decided to merge.